Header header
string observer_name
string target_name
humanoid_models/SoftClassification[] classifications
